src/test/java/org/springframework/data/elasticsearch/core/index/MappingBuilderIntegrationTests.java

```diff
@@ -77,6 +78,12 @@ void cleanup() {
 		operations.indexOps(IndexCoordinates.of(indexNameProvider.getPrefix() + "*")).delete();
 	}
 
+	@Test
+	public void shouldSupportAllTypes() {
```

Comment:
This one is not strictly related but `EntityWithAllTypes` was not used in tests at all, fixing that